src/controllers/check.your.answers.controller.ts

```diff
@@ -1,16 +1,25 @@
+import { Session } from "ch-node-session-handler";
 import { NextFunction, Request, Response } from "express";
 import ObjectionCompanyProfile from "../model/objection.company.profile";
 import { Templates } from "../model/template.paths";
+import { Objection } from "../modules/sdk/objections";
+import { getObjection } from "../services/objection.service";
 import { retrieveCompanyProfileFromObjectionSession } from "../services/objection.session.service";
 import logger from "../utils/logger";
 
-export const get = async (req: Request, res: Response, next: NextFunction) => {
+export const get = async (req: Request, res: Response, next: NextFunction): Promise<void> => {
```

Comment:
you should just be able to declare the return type as void, and as it's an async function, will automatically wrap the result in a Promise.

Reply:
saying that, the default function return type is void/undefined anyway

Reply:
So do you think it needs changing then?

Reply:
personally, i would have left it as the original version. If you want to be more explicit, : void should be fine

Reply:
Ok, I'll revert.